### DISCUSSION TITLE
Advertise release :)

### DISCUSSION THREAD

**Discussion by jorgecarleitao** [DataEngineeringLabs/arrow2-convert#31] [Ideas]
@ncpenke ; some random thoughts:

* I usually post on reddit when I release a new crate; however, I feel that you should be the one taking the credit. Would you be interested in posting it on r/rust?
* We could also write a small blog post together to show-case what we can do with it.
* We can also link this crate on arrow2's README. 

Any thoughts?

**Comment by ncpenke** [comment]
@jorgecarleitao That would be amazing!

As an aside I'm almost done with enum/union support (I actually have a question around how unit variants are encoded which we can discuss in the PR). I think we might as well wait until that's merged before we advertise since it should be a more complete set of functionality.

I'll post a blurb here before posting on r/rust so that you can review. Please let me know how you want to collaborate on the blog post. Thanks!

**Reply by jorgecarleitao** [reply]
savage! 🚀 Well, let's wait them. Maybe I can release arrow2 0.10 meanwhile and we can point to it.

**Comment by ncpenke** [comment]
@jorgecarleitao Congrats on the arrow2 0.10.0 release! I'm at a good checkpoint for enums/unions, but just trying to figure out whether we need support for scalars for the deserialization path. It can be done without, but I think the cleanest way is actually to use the arrow2::UnionIter. 

I'm wrapping a few things that are time critical for other projects but planning to get back and wrap this up in the next couple of weeks.
